test/Tereshchenko_Igor/laboratory1.sql

```diff
@@ -1,2 +1,86 @@
--- LABORATORY WORK 1
--- BY Tereshchenko_Igor
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+модифікувати таблиці та вставляти дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Sedinin
+IDENTIFIED BY Pass
+USED TABLESPACE "USER"
+TEMPORARY TABLESPACE "TEMP"
+
+GRANT "CONNECTION" TO Sedinin
+
+GRANT ALTER ALL TABLES TO Sedinin
+GRANT INSERT ALL TABLES TO Sedinin;
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент купляє квиток на потяг.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE TABLE Student (
+  student_name char(30) NOT NULL )
+ALTER TABLE Student 
+  ADD CONSTRAINT student_name_pk PRIMARY KEY (student_name);
+  
```

Comment:
student_name VARCHAR2(20) NOT NULL,
 /[^[A-Z]{1}[a-z]{1,9}$/

student_surname VARCHAR2(20) NOT NULL,
 /[^[A-Z]{1}[a-z]{1,9}$/

birthday VARCHAR2(10) ,
 /^[0-9]{2}[.]{1}[0-9]{2}[.]{1}[012]{1}[0-9]{4}$/

sex VARCHAR2(1) ,
 /^[m}F] {1}$/ (M - male, F -female)